pps/scraper.py

```diff
@@ -1,21 +1,82 @@
-import logging
+# encoding=utf8
 
+import logging
+import sys
+import requests
 from cliff.command import Command
+from bs4 import BeautifulSoup
+from html2text import html2text as gauss
+
 
 
 class Scraper(Command):
-    "A simple command that prints a message."
 
     log = logging.getLogger(__name__)
 
+    def get_parser(self, prog_name):
+        parser = super(Scraper, self).get_parser(prog_name)
+        parser.add_argument('-u', '--url', type=str)
+        parser.add_argument('-o', '--output', type=str)
+        return parser
+
     def take_action(self, parsed_args):
-        self.log.info('test')
+        argx = sys.argv[1:]
+        self.log.info('testing')
         self.log.debug('debugging')
-        self.app.stdout.write('0\n')
+        
+        option = str(parsed_args.url)
+        self.log.info('Arguments: '+str(argx)+'\n')
+        self.log.info(option+'\n')
```

Comment:
I think infix arguments like `+` should have spaces around them. You do this in most places in your code, but not here.